src/main/java/org/isf/medicalinventory/manager/MedicalInventoryManager.java

```diff
@@ -211,4 +211,13 @@ private void validateMedicalInventory(MedicalInventory medInventory) throws OHDa
 			throw new OHDataValidationException(errors);
 		}
 	}
+
+	/**
+	 * Marks an inventory as deleted by changing its status.
+	 * @param medicalInventory - the medicalInventory of the inventory to delete.
+	 * @throws OHServiceException if an error occurs during the operation.
+	 */
+	public void deleteInventory(MedicalInventory medicalInventory) throws OHServiceException {
+		ioOperations.deleteInventory(medicalInventory);
```

Comment:
I think we should also delete new lots created, wdyt @ArnaudFonzam ?

Reply:
absolutely @mwithi